cpp/src/runtime/context.cpp

```diff
@@ -30,12 +30,17 @@ context::context() : _program_cache{std::make_unique<jit::program_cache>()}
 {
   auto dump_codegen_flag = getenv_or("LIBCUDF_JIT_DUMP_CODEGEN", std::string{"OFF"});
   _dump_codegen          = (dump_codegen_flag == "ON" || dump_codegen_flag == "1");
+
+  auto use_jit_flag = getenv_or("LIBCUDF_USE_JIT", std::string{"OFF"});
```

Comment:
I'd prefer this name:
```suggestion
  auto use_jit_flag = getenv_or("LIBCUDF_JIT_ENABLED", std::string{"OFF"});
```